pkg/components/server.go

```diff
@@ -17,6 +18,15 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
+const (
+	initialProbesDelaySeconds = 90
+
+	probesTimeoutSeconds = 3
+	probesPeriodSeconds  = 15
+
+	readinessProbeHTTPPath = "/solomon/status"
```

Comment:
solomon path shouldn't be hardcoded in the public code, it should be configured in spec or somehow else.

Reply:
Is there any specific need to use solomon?
I think it is ok to use host:monitoring_port/orchid/service, it is just fine. And I do not see any issue to hardcode this particular address in the code, it is a detail of YT's implementation, which is shared by absolutely all C++ server components.